website/docs/concepts/providers.mdx

```diff
@@ -96,34 +96,31 @@ void main() {
 
 :::
 
-## Performing actions before the state destruction
```

Comment:
Why is this removed?

Reply:
This got moved to the provider lifecycle page.